spec/factories/users.rb

```diff
@@ -6,5 +6,9 @@
     first_name { 'John' }
     last_name { 'Doe' }
     password { 'password-1234' }
+
+    trait :confirmed do
+      email_confirmed_at { DateTime.now }
```

Comment:
Why not use `Time.current` here, if not for consistency?

Reply:
Using `DateTime.now` in the model.